containerize/process_log.go

```diff
@@ -0,0 +1,100 @@
+package containerize
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"sync"
+	"teleport-exec/filestream"
+)
+
+//-------------------------------------------------------------------------------------------------
+type ProcessLog struct {
+	command_id  string
+	fd          *os.File
+	readers     map[*filestream.FileStream]bool
+	readersLock sync.Mutex
+}
+
+func NewProcessLog(command_id string) (*ProcessLog, error) {
+	file_name := "/tmp/command-" + command_id + ".out"
+
+	fd, err := os.Create(file_name)
```

Comment:
I would use the tools in the `os` package for making temporary dirs/files. This will make sure you leverage the appropriate temp directory for the system.

Reply:
good call, it is definitely better